uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/PublicHolidayLeaveRequestCreation.php

```diff
@@ -160,16 +160,18 @@ private function create($contactID, AbsenceType $absenceType, PublicHoliday $pub
   private function createLeaveRequest($contactID, AbsenceType $absenceType, PublicHoliday $publicHoliday) {
     $leaveRequestStatuses = array_flip(LeaveRequest::buildOptions('status_id', 'validate'));
     $leaveRequestDayTypes = array_flip(LeaveRequest::buildOptions('from_date_type', 'validate'));
+    $publicHolidayDate = new DateTime($publicHoliday->date);
+    $publicHolidayDate->setTime(00, 00, 00);
```

Comment:
Tried that but was failing for some tests that already pass time in addition to date when creating a public holiday leave request. Left it as is instead of looking for all the tests and changing them.